packages/legend-manual-tests/src/__tests__/roundtrip-grammar/RoundtripGrammar.test.ts

```diff
@@ -83,6 +83,7 @@ const EXCLUSIONS: { [key: string]: ROUNTRIP_TEST_PHASES[] | typeof SKIP } = {
     ROUNTRIP_TEST_PHASES.PROTOCOL_ROUNDTRIP,
     ROUNTRIP_TEST_PHASES.GRAMMAR_ROUNDTRIP,
   ],
+  'DSL_DataSpace-mapping-include-dataspace.pure': SKIP,
```

Comment:
why are we skipping this?

Reply:
I should be able to add this back when I address your other comment re the 'support the unsupported'